.github/workflows/macos.yml

```diff
@@ -43,11 +43,11 @@ jobs:
         export CXX=g++-6
         mkdir gcc_aliases
         pushd gcc_aliases
+        # make gcc aliases in current dir
         ln -s `which gcc-6` gcc
         ln -s `which g++-6` g++
         ln -s `which gfortran-6` gfortran
-        # make gcc aliases in current dir
-        export PATH=$PWD/gcc_aliases:$PATH
+        export PATH=$PWD:$PATH
```

Comment:
Github actions is based on azure, right? On azure, each script runs in its own environment so `export`ed variables doesn't persist across scripts.  You should be able to define variables in the yaml that cover the whole job though. In azure it would be:
```
variables: 
  FC: gfortran-6
  CC: gcc-6
  CXX: g++-6
```

Reply:
Looks like gihub actions use the name [`env`](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#about-environment-variables).

I don't know why this script uses the `-6` suffix for the commands defined by the environment variables `FC`, `CC` and `CXX`, but then names the executables `gfortran`, `gcc` and `g++` (without the `-6`) in the directory that is added to $PATH.

Reply:
Probably a hack because the environment variable wasn't being picked up? Comments suggest this was translated from travis, where the env variables would be visible across steps.

Reply:
@andyfaff Any ideas?

Reply:
I'll try setting `env` for the entire jobs, and remove all the `gcc_aliases` stuff.  (Just waiting for the current tests to fail as exepected.)